### ISSUE TITLE
`window.location.reload()` doesn't work; `window.location.href = window.location.href;` doesn't reload the page

### ISSUE THREAD

**Issue by shawwn** [exokitxr/exokit#1232]
**Describe the bug**
What happened, and what did you expect to happen?

After navigating to a valid url (e.g. `window.location.href = 'https://ga-nksuawgghv.now.sh/'` in the CLI), there doesn't seem to be any way to refresh the page. `window.location.reload()` doesn't seem to do anything, nor does `window.location.href = window.location.href`

**Comment by KingNelx** [comment]
yes same thing happen on me. What i did is just location.reload and it works
